working-together/README.md

```diff
@@ -0,0 +1,23 @@
+Working Together
+==============
+
+A guide for working well together.
+
+
+Pair Programming
+-------
+
+**Before:**
+* Agree Scope (Pairing for two hours? Until the ticket is complete? Just get pass the bug?)
+* Agree Working Environment (Two keyboards? VIM/what other tools?)
```

Comment:
Even though I love `vim`, I don't think everyone uses it in the office. What do you think about changing this to "Text editor and other tools?"